content/blog/3-same-subject-continued-concerning-dangers/index.md

```diff
@@ -0,0 +1,45 @@
+
+For this essay, I am only considering safety as it relates to maintaining peace from foreign violence and domestic violence. I'll start with foreign violence. So, let's think about whether people are right that a national government really affords the best security against hostilities from abroad.
+
+The number of wars in the world is proportionate to the number and weight of real or pretended causes. If this is right, then we should think about whether a united America or a divided America is likely to produce more just causes for war; if it turns out that a united America will give fewer causes for war, then a union will be more likely to lead to peace with other nations.
+
+So, what are the just causes for war? They typically come either from broken treaties or direct violence. America has already formed treaties with six countries--and all but Prussia have strong navies (and so they could attack us). We also have lots of trade with Portugal, Spain, and Britain, which all have colonies on our continent. <!-- This is a pretty big paraphrase -->
+
+In order to have peace, we need to be sure to observe international laws and norms with regard to these powerful countries, and it seems obvious to me that this will be easier and more effective if done by a single national government rather than by thirteen separate states or a few confederacies.
+
+Once an efficient national government is established, the best people in the country will naturally serve there. Because we can select from the entire country, we will never have the problem that some states have where they can't really find anyone qualified and the cream of the crop will serve in the national government <sup>[1](#footnote1)</sup>. The administration, political bodies, and judicial decisions of a national government will, therefore, be more wise, well-reasoned, and judicious than those of individual states. Having this group of our best people in charge of our national direction will lead to better relations with other countries, and therefore more safety for us.
+
+Our relationship with these other countries will require making and executing treaties. Can you imagine trying to enforce treaties with thirteen different, independent court systems, with judges appointed by different, independent governments! 🤮 The members of the Constitutional Convention were wise to commit these sorts of questions to a single, national court.
+
+Indeed, sometimes the governing party of a single state might be tempted to gain an advantage by breaking a treaty. However, a single state won't have enough power to cause a national government to make these sorts of bad decisions, and justice will be preserved. See the peace treaty with Britain for an example of how important this is. <!-- Worth looking into what he's talking about -->
```

Comment:
I can't find which peace treaty with Britain he's talking about. Maybe the Treaty of Paris? I think if we can't get any more context, we can cut that sentence.

Reply:
I think it's the Treaty of Paris - it sounds like, e.g., some states didn't live up to the treaty (like restoring property to Loyalists) - https://en.wikipedia.org/wiki/Treaty_of_Paris_%281783%29

Happy to defer to you as to whether to keep/expand/delete.